__tests__/reservations-counter.test.js

```diff
@@ -0,0 +1,24 @@
+import { resCounter } from '../src/modules/reservations.js';
+
+describe('the reservations counter function', () => {
```

Comment:
Great Job @rskinnerc , good describing test.

You can mark your work as done!.